src/components/v2/Icon/icons/sort.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="12" height="12" viewBox="0 0 12 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M2 7.71411L6.36358 11.9998L10.7272 7.71411H2Z" fill="white"/>
```

Comment:
and in all other icons

Reply:
In this case, I figured that since we can't change the colour of the highlighted caret (on the `sortUp` and `sortDown` icons), then we might have to use these icons as is.
What do you think?

Reply:
yep. I think we can edit if it would be needed in future